src/rust/engine/fs/store/src/lib.rs

```diff
@@ -73,6 +75,9 @@ use crate::remote::ByteStoreError;
 const MEGABYTES: usize = 1024 * 1024;
 const GIGABYTES: usize = 1024 * MEGABYTES;
 
+/// How big a file must be to become an immutable input symlink.
+const IMMUTABLE_FILE_SIZE_LIMIT: usize = MEGABYTES;
```

Comment:
What's the best way to find a "reasonable" limit? 1MB was just a pick out of a hat.